src/components/sidenav/sidenav.js

```diff
@@ -365,6 +367,8 @@ function SidenavDirective($mdMedia, $mdUtil, $mdConstant, $mdTheming, $animate,
       ]).then(function() {
         // Perform focus when animations are ALL done...
         if (scope.isOpen) {
+          // Notifies child components that the sidenav was opened.
+          ngWindow.triggerHandler('resize');
```

Comment:
So after the SideNav open animation finishes, you fire a `resize` event that causes reflows for the entire window?